contentcuration/contentcuration/views/users.py

```diff
@@ -51,7 +51,7 @@ def send_invitation_email(request):
         }
 
         # Need to break into two steps to avoid MultipleObjectsReturned error
-        invitation = Invitation.objects.filter(**fields).first()
+        invitation = Invitation.objects.filter(channel_id=channel_id, email=user_email).first()
```

Comment:
If a user registers (i.e setting his or her name) and the original inviter invites them again, it will be considered a separate invitation, making the user invited twice